### ISSUE TITLE
`fwupd` overy verbose startup errors that are not errors.

### ISSUE THREAD

**Issue by ahkok** [clearlinux/distribution#212]
```
fwupd[268]: disabling plugin because: failed to startup dell_esrt: UEFI firmware already supported
fwupd[268]: disabling plugin because: failed to startup dell: Firmware updating not supported
fwupd[268]: disabling plugin because: failed to startup redfish: no SMBIOS table
fwupd[268]: disabling plugin because: failed to coldplug amt: ME refused connection
fwupd[268]: disabling plugin because: failed to coldplug thunderbolt_power: No support for force power via kernel or bolt
```

None of these are relevant to any user, and they should be removed from the logs.

**Comment by ahkok** [comment]
@bryteise

**Comment by bryteise** [comment]
So this is difficult because the plugin setup errors seem to only have one error type. If you want us to carry a patch making any failure in this path a debug log level that would probably be as close as you can get. I do note in defense of these errors, it is very useful when things that should work do not. Figuring out that firmware you have isn't being checked isn't particularly obvious without these log messages and knowing to first tweak the service to enable debug first isn't obvious either.

**Comment by tpleavitt** [comment]
@ahkok , please re-evaluate this issue

**Comment by superm1** [comment]
FYI, current versions of fwupd are not this verbose anymore.

**Comment by superm1** [comment]
This can be closed, definitely confirmed in 1.3.3 not a problem.
